app/src/main/java/org/jellyfin/androidtv/util/profile/ProfileHelper.kt

```diff
@@ -143,12 +143,16 @@ object ProfileHelper {
 	fun addAc3Streaming(profile: DeviceProfile, primary: Boolean) {
 		if (Utils.downMixAudio()) return
 
-		val mkvProfile = findTranscodingProfile(profile, ContainerTypes.MKV) ?: return
-
-		Timber.i("*** Adding AC3 as supported transcoded audio")
-		mkvProfile.audioCodec = when (primary) {
-			true -> "${CodecTypes.AC3},${mkvProfile.audioCodec}"
-			false -> "${mkvProfile.audioCodec},${CodecTypes.AC3}"
+		val containers = arrayOf(ContainerTypes.MKV, ContainerTypes.TS)
+		containers.forEach { c ->
+			Timber.d("trying to match transcoding profile for %s", c)
+			val transcodingProfile = findTranscodingProfile(profile, c) ?: return@forEach
+
+			Timber.i("*** Adding AC3 as supported transcoded audio for %s", c)
+			transcodingProfile.audioCodec = when (primary) {
+				true -> "${CodecTypes.AC3},${transcodingProfile.audioCodec}"
+				false -> "${transcodingProfile.audioCodec},${CodecTypes.AC3}"
+			}
```

Comment:
Looking at the usages of `addAc3Streaming` the primary argument is always true so this function can be simplified a bit.

Reply:
In fact, it's only used for exoplayer profiles so we might just pass a boolean to the exoplayerprofile constructor whether to enable or disable ac3 and remove this helper function.

https://github.com/jellyfin/jellyfin-androidtv/blob/8424e581583743d7d0a09902716defbfde875bc7/app/src/main/java/org/jellyfin/androidtv/ui/playback/PlaybackController.java#L498-L502

Reply:
`UserPreferences.Companion.getAc3Enabled()` looks like it just resolves to !DeviceUtils.isFireTvStickGen1()
https://cs.github.com/jellyfin/jellyfin-androidtv?q=pref_bitstream_ac3

So anything using exoplayer profile will qualify, and ac3 might as well just be added to the profile by default?

Reply:
The preference can still be set in the app. It's disabled by default on fire tv.